modules/core/src/main/scala/gem/Dataset.scala

```diff
@@ -49,12 +50,12 @@ object Dataset {
      * @group Typeclass Instances
      */
     implicit val LabelOrder: Order[Label] =
-      Order[Observation.Id].contramap[Label](_.observationId) |+|
+      Order[Observation.Id].contramap[Label](_.observationId) whenEqual
       Order[Int]           .contramap[Label](_.index)
 
     /** @group Typeclass Instances */
     implicit val LabelShow: Show[Label] =
-      Show.showA
+      Show.fromToString
```

Comment:
Yeah the semigroup for `Eq` isn't implicit for some reason. I talked to Kai and we can't see a reason for this so it may be changed. But yeah in any case `.whenEqual` is fine.

I think the `.fromToString`, etc., are a bit nicer, yeah. Cats is more thoughtful about naming and much more consistent.